core/lib/protobuf_config/src/proto/config/api.proto

```diff
@@ -11,9 +11,7 @@ message MaxResponseSizeOverride {
 
 message Web3JsonRpc {
   optional uint32 http_port = 1; // required; u16
-  optional string http_url = 2; // required
   optional uint32 ws_port = 3; // required; u16
-  optional string ws_url = 4; // required
```

Comment:
That's a bit of a controversial change; on the one hand, it's not required for running the server. But it does help for running other tools against the server